src/service/config.proto

```diff
@@ -4,32 +4,53 @@ package helium.config;
 
 import "region.proto";
 
+// ------------------------------------------------------------------
+// Message Definitions
+// ------------------------------------------------------------------
+
+// - Every message including `signature` will need to be signed over the all
+//    message, with the `signature` field set to an empty value.
+// - Every `timestamp` is in millseconds
+
+enum action_v1 {
+  create = 0;
+  update = 1;
+  delete = 2;
+}
+
+// Define an organisation
 message org_v1 {
   uint64 oui = 1;
   bytes owner = 2;
   bytes payer = 3;
   uint32 nonce = 4;
 }
 
+// Device address range, ex: 16#00000001 to 16#0000000A
 message devaddr_range_v1 {
   uint64 start_addr = 1;
```

Comment:
Maybe we want to change these addrs to `uint32`.
That covers the entire devaddr range exactly.